gtsfm/runner/gtsfm_runner_base.py

```diff
@@ -127,6 +127,25 @@ def construct_argparser(self) -> argparse.ArgumentParser:
             default=None,
             help="tmp directory for dask workers, uses dask's default (/tmp) if not set",
         )
+        parser.add_argument(
+            "--run_cluster",
+            action="store_true",
+            default=False,
+            help="whether to run gtsfm on a cluster"
+        )
+        parser.add_argument(
+            "--cluster_config_name",
+            type=str,
+            default="cluster.yaml",
+            help="config listing IP worker addresses for the cluster,"
+            " first worker is used as scheduler and should contain the dataset",
+        )
```

Comment:
Can we run on a single machine using a yaml file too?

Reply:
Yes, I will be creating a config file to keep all the parameters (including the option to run on a single machine) in a separate PR.